package.json

```diff
@@ -82,13 +83,14 @@
     "minimist": "^1.2.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.7.1",
-    "puppeteer": "^10.0.0",
-    "rollup": "~2.38.5",
+    "pug": "^3.0.1",
```

Comment:
Why add `pug` dep?

Reply:
We don't use `require.resolve('@vue/consolidate/package.json', { paths: [packageDir] })` now.

`@vue/consolidate/package.json` in rootDir depends on `"pug"`  in rootDir.